hicexplorer/hicPlotDistVsCounts.py

```diff
@@ -384,17 +385,13 @@ def main(args=None):
             axs[row, col] = ax
             idx += 1
             if args.outFileData is not None:
-                if args.perchr and len(args.matrices) > 1:
-                    label = labels[matrix_file]
-                    args.outFileData.write("#{}\n".format(chrom))
-
-                elif args.perchr:
-                    label = chrom
-                else:
-                    label = labels[matrix_file]
-                args.outFileData.write("#{}\n".format(label))
-                args.outFileData.write("\t".join(map(str, x)) + "\n")
-                args.outFileData.write("\t".join(map(str, y)) + "\n")
+                x_vals = np.stack(x).T
+                y_vals = np.stack(y).T
+                table_to_export = pd.DataFrame({'Matrix': matrix_file,
+                                                'Chromosome': chrom,
+                                                'Distance': x_vals,
+                                                'Contacts': y_vals})
+                table_to_export.to_csv(args.outFileData, sep='\t')
```

Comment:
I think is better to remove the first column (the index column when saving). Just add `index=False` to the `.to_csv` function.

Reply:
Hum I'm not so sure, because when you plot multiple chromosomes / matrices, having the indexes help to identify when a given table is starting / ending inside the file (the different tables are all on top of each other in the output file).